pydantic_ai_slim/pydantic_ai/__init__.py

```diff
@@ -43,6 +43,7 @@
     'RunContext',
     # result
     'ToolOutput',
+    'StructuredOutput',
```

Comment:
I am not necessarily opposed to this choice of name, but I feel like the term "Structured Output" is often used to refer to the more general idea, which includes output produced via tool calls and output produced using `response_format` etc.